### DISCUSSION TITLE
Odd, non-existent Entities

### DISCUSSION THREAD

**Discussion by GrizzlyAK** [zwave-js/zwave-js#5884] [General]
I've noticed this since way back using OZW v1.4, and it still happens today, and I'm not sure what causes it. In HA, I have a LOT of ZW devices and tons of associated entities. It seems every time I look at my entities list (which isn't that often anymore), I find some new, obscure entities that weren't there before, and nearly always don't exist in the device, or is not even related to the device type. The latest involves the Zooz ZEN21 in-wall light switch. Not much else to it other than that, but HA is showing me the following Entities in the Entities list:

- Humidity
- Illuminance
- Motion Detection

Granted, these are usually marked with a "Restored" status, but why are they even there? Inquiring minds want to know. 

Although, now that I think about it, I think it is HA who creates the entities, so maybe I need to bring this up with them?

Thanks,
Shane

**Comment by AlCalzone** [comment]
If the devices are included using Security S0 or S2, this is most likely something in HA.

If not, you're most likely receiving corrupted frames and [this comment](https://github.com/home-assistant/core/issues/77966#issuecomment-1449797663) explains what happens.
